lib/ain-evm/src/transaction/mod.rs

```diff
@@ -300,6 +300,14 @@ impl SignedTx {
             TransactionV2::EIP1559(tx) => tx.hash(),
         }
     }
+
+    pub fn get_tx_type(&self) -> U256 {
+        match &self.transaction {
+            TransactionV2::Legacy(_) => U256::from(0),
+            TransactionV2::EIP2930(_) => U256::from(1),
+            TransactionV2::EIP1559(_) => U256::from(2),
+        }
+    }
 }
```

Comment:
TransacationV2 already has a type_id() method

Reply:
Resolved in commit [70c035a](https://github.com/DeFiCh/ain/pull/2382/commits/70c035ad71c3024bc8ddfc755a8e6bfc8ff9e940)